src/AccordionPanelGroup.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import cx from 'classnames';
+
+export const ACCORDIONPANELGROUP_CLASS = 'accordionPanelGroup';
+/**
+ * @module AccordionPanelGroup
+ */
+class AccordionPanelGroup extends React.Component {
+	constructor(props) {
+		super(props);
+		this.accordionPanels = this.props.accordionPanels.map((accordionPanel, i) => {
+			return this.clonePanels(accordionPanel, i, accordionPanel.props.isOpen);
+		});
+
+		this.state = {
+			clickedPanel: null
+		};
+
+		this.clonePanels = this.clonePanels.bind(this);
+		this.setClickedPanel = this.setClickedPanel.bind(this);
+	}
+
+	/**
+	 * @param clickedPanel is the `AccordionPanel` component passed in from `_handleToggle`
+	 * @returns {undefined}
+	 *
+	 * Keeps track of the clicked panel in order to support AccordionPanelGroups that only
+	 * have one panel open at a time.
+	 */
+	setClickedPanel(clickedPanel) {
+		this.setState({ clickedPanel });
+	}
+
+	/**
+	 * @param {Object} accordionPanel - `AccordionPanel` components to clone
+	 * @param {number} i - index of the `AccordionPanel`
+	 * @param {boolean} isOpen - whether the `AccordionPanel` is open or not
+	 * @returns {Array} `AccordionPanel` components with props from `AccordionPanelGroup`
+	 */
+	clonePanels(accordionPanel, i, isOpen) {
+		const panelProps = {
+			key: i,
+			iconAlign: this.props.iconAlign,
+			iconSize: this.props.iconSize,
+			iconShape: this.props.iconShape,
+			iconShapeActive: this.props.iconShapeActive,
+			isAnimated: this.props.isAnimated,
+			className: accordionPanel.props.className,
+			setClickedPanel: this.props.multiSelectable ? false : this.setClickedPanel,
+			isOpen: isOpen
+		};
+		return React.cloneElement(accordionPanel, panelProps);
+	}
+
+	/**
+	 * @returns {Array} `AccordionPanel` components with the correct value for `isOpen` prop
+	 */
+	renderAccordionPanels() {
+		this.accordionPanels = this.accordionPanels.map((accordionPanel, i) => {
```

Comment:
pretty sure this could also be simplified to `this.accordionPanels = this.accordionPanels.map(this.clonePanels)`